lib/segment/src/index/field_index/histogram.rs

```diff
@@ -179,6 +176,79 @@ impl<T: Numericable> Histogram<T> {
         Unbounded
     }
 
+    /// Infers boundaries for bucket of given size and starting point, skipping the first
+    /// border next to the starting point.
+    ///
+    /// E.g:
+    ///                                starts counting in this border
+    ///                                    ---> v
+    /// 1, 2, 3, 4, 5, 5, 5, 5, 6, 7, 8, 9, 10, 11, 12, 13
+    /// b           [                    b      b       b
```

Comment:
What's `b`?

Reply:
I haven't looked into the details yet, but maybe you can denote it with `|`.

Reply:
Not needed anymore